kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KopEventManager.java

```diff
@@ -61,7 +62,7 @@
             new LinkedBlockingQueue<>();
     private final KopEventThread thread =
             new KopEventThread(kopEventThreadName);
-    private final GroupCoordinator coordinator;
+    private final Map<String, GroupCoordinator> groupCoordinatorsByTenant;
```

Comment:
There are no need to create a `KopEventManager` for each `GroupCoordinator`, but pass the `GroupCoordinator` list to `KopEventManager`